website/astro.config.mjs

```diff
@@ -9,7 +9,6 @@ export default defineConfig({
   // setup for github pages
   site: 'https://proxy-memoize.js.org',
   // note: this is the path to the repo, not the path to the site
-  base: '',
```

Comment:
wouldn't it be nice to keep it to run it locally?

Reply:
I see. It's already working locally, why would we keep it?

Reply:
You are right. It's actually not necessary to work locally. I just would be comfortable to keep it in case it's used in a different `base`.